CONTRIBUTING.md

```diff
@@ -352,46 +362,47 @@ After copying release notes, use `make package` to create the package file (e.g.
 $ pecl install -f mongodb-X.Y.Z.tgz
 ```
 
-### Commit version update and release notes
+### Update version info
 
-Commit the modified `phongo_version.h` file as "Package X.Y.Z"
+Commit the modified `phongo_version.h` file and push this change:
 
 ```
 $ git add phongo_version.h
 $ git commit -m "Package X.Y.Z"
+$ git push mongodb
 ```
 
-### Tag release
+> **Note:** Pushing this commit independently from the subsequent "Back to -dev"
+> commit will ensure that Windows build artifacts are created for the release.
```

Comment:
GitHub Actions are smart enough to not build individual commits if multiple are pushed at once. The original example we had where "Package X.Y.Z" _and_ "Back to -dev" were pushed simultaneously would have meant we never generate build artifacts for the actual tag.

This is clunky and definitely something we could improve down the line if we stop depending on build artifacts from a previous CI run and instead build DLLs on-demand for releases. We could also avoid debug builds and PDB file generation for typical CI runs (I imagine that's a bit slower than without).

Reply:
There is the option of pushing them at the same time, then pushing the tag separately and ensuring the tag is built. The workflow syntax allows you to specify tag filters to build on a push: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags.

If we're updating the process down the line, we could change the process to build tags and create a draft release for them, attaching build artifacts in the process. We could then revise the release notes separately. For now I don't think we need to change this, as I'm sure we'll be working on automating releases (including PECL packages) in the near future.

Reply:
Noted. I didn't consider that since we currently only have branch filters in place.

If we revisit release automation down the line I'd definitely suggest writing up a design doc first so we can plan out exactly what we want to implement.

Reply:
I agree, that change exceeds the scope of getting this to work. I suggest putting release automation on our quarterly plan and designing this ahead of time 👍